### DISCUSSION TITLE
Incorporate Troubleshoot.md

### DISCUSSION THREAD

**Discussion by ravinder-tw** [NVIDIA-NeMo/Guardrails#805] [General]
Hello Folks, 

I recently started to work on this repo and feel it will be worth adding `Troubleshoot.md` file into this repo. 

I myself had to do couple of workarounds to get the examples and repo up and running, WDYT?

cc: @Pouyanpi

**Comment by Pouyanpi** [comment]
Hi @ravinder-tw, that sounds great! It would be helpful if you could start by opening an issue and then follow up with a PR. This way, others who are interested can contribute as well. I’m looking forward to seeing this come together!
